src/features/opportunites/pages/OpportuniesList/DeleteConfirmation.tsx

```diff
@@ -0,0 +1,25 @@
+import { BaseModal, ModalHeader } from "../../../../components/Modal/modals";
+import { Box, Button, Typography } from "@mui/material";
+
+interface DeleteConfirmationDialogProps {
+  open: boolean;
+  onClose: () => void;
+  onConfirm: () => void;
+}
+
+export function DeleteConfirmationDialog({ open, onClose, onConfirm }: DeleteConfirmationDialogProps) {
+  return (
+    <BaseModal open={open} onClose={onClose}>
+      <ModalHeader title="Confirm Deletion" onClose={onClose} />
+      <Typography sx={{ mt: 2 }}>
+        Are you sure you want to delete this item?
+      </Typography>
+      <Box sx={{ display: "flex", justifyContent: "flex-end", mt: 3 }}>
+        <Button onClick={onClose}>Cancel</Button>
+        <Button onClick={onConfirm} color="error">

```

Comment:
You can use somthing like this to make the modal more responsive

<BaseModal extraBaseStyles={{ maxWidth: "900px", width: "95%" }}  />

Note: configure the maxWdith base on your requirements